lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -476,7 +476,7 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
-  if (typeName == "MatMul" || typeName == "BatchMatMul") {
+  if (typeName == "MatMul") {
```

Comment:
You can unify loading of MatMul and BatchMatMul by introducing a flag `isBatched` to `loadBatchMatMul()` method, and calling `loadBatchMatMul(..., isBatched=false)` method from here. In other words, regular `MatMul` won't fall under common ops, but will still share code.